pkg/vdbgen/vdb.go

```diff
@@ -475,6 +551,37 @@ func (d *DBGenerator) setSubclusterDetail(ctx context.Context) error {
 	return nil
 }
 
+// setImage will fetch the server version and use it to build
+// and set the image inside v.vdb.
```

Comment:
Suggest the following.
```suggestion
// setImage will fetch the server version and use it to pick
// an image that is hosted on our docker repository.
```